### PR TITLE
doc(ds/complexity): Spell checks

### LINKED ISSUE #23: Spell Checks on docs

### PR DESCRIPTION
**Pull request by sriramanvellingiri**
#23

### CODE REVIEW

File: ComplexityAnalysis/SpaceComplexity.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Space complexity = Input Size + Auxillary Space.
 
 ## To design the algorithm
  
-Time complexity much be lower, where the space complexity can be higher.
+ Time complexity is inversely proportional to the space.so the time complexity can be lower if the space complexity is higher.
```
Comment (shangan23) [inline ComplexityAnalysis/SpaceComplexity.md]: change "Time complexity is inversely proportional to space" to "Time complexity is inversely proportional to space complexity."

Comment (shangan23) [inline ComplexityAnalysis/SpaceComplexity.md]: Change the second sentence to start with the Capital case and give a space after '.' like '. So'

**Review by shangan23 — CHANGES_REQUESTED** [review]
Please check the review comments.

File: ComplexityAnalysis/SpaceComplexity.md (hunk 1 above)
Comment (sriramanvellingiri) [reply ComplexityAnalysis/SpaceComplexity.md]: It is resolved

**Pull request merged**